docs/guides/admin/docs/upgrade.md

```diff
@@ -48,6 +48,20 @@ Please make sure to compare your configuration against the current configuration
   [this document](https://github.com/elan-ev/opencast-studio/blob/2020-09-14/CONFIGURATION.md).
 
 
+### Wowza streaming configuration changes
+
+In [pull request #1179](https://github.com/opencast/opencast/pull/1179), the configuration of Wowza servers was made
+tenant-specific. Before, you had to specify `etc/org.opencastproject.streaming.url` in `custom.properties`. Starting
+with Opencast 9, that value has to specified as `org.opencastproject.mh_default_org.wowza.url` in
+`org.opencastproject.distribution.streaming.wowza.WowzaStreamingDistributionService.cfg`.
+
+Also as part of that pull request, support for RTMP was dropped. This shouldn't be a problem in most cases, however. If
+your streaming URL started with `rtmp://`, try changing it to `https://`. In most cases, RTMP wasn't even used before,
+despite it being specified in the URL. Instead, the formats/protocols that can be used are specified in
+`org.opencastproject.wowza.formats` at the top of the same file.
```

Comment:
Maybe this could be a bit more concise?

Reply:
Mhhhh what about this?

> Additionally, support for RTMP was dropped. This shouldn't be a problem in most cases, as RTMP usually wasn't used before, despite being specified in the URL. If your streaming URL started with `rtmp://`, try changing it to `https://`. Streaming formats/protocols can be specified in `org.opencastproject.wowza.formats` at the top of the same file.

Reply:
I'd scratch the last line since that isn't a new thing (the key just got renamed and RTMP removed). But otherwise, yeah, sounds good!